site/content/blog/2021-09-01-whats-new-in-svelte-september-2021.md

```diff
@@ -0,0 +1,80 @@
+
+If you'd like to help, please consider working on any of the [1.0 milestone issues](https://github.com/sveltejs/kit/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0).
+
+Some noteable changes in SvelteKit in the last month include...
+
+- SvelteKit will now detect if a prerendered app is trying to access a query paremter and return an error - instead of silently failing ([#2104](https://github.com/sveltejs/kit/pull/2104))
+- create-svelte's final output dir has been standardized as `/build` (vs `/.svelte-kit`) ([#2109](https://github.com/sveltejs/kit/pull/2109))
+
+To see all updates to SvelteKit, check out the [SvelteKit changelog](https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md).
+
+
+---
+
+## Community Showcase
+
+**Apps & Sites**
+- [macos-web](https://github.com/PuruVJ/macos-web) by @purubjdeb has been rebuilt with Svelte from the ground up. Check out all the detials in this [Twitter thread](https://twitter.com/puruvjdev/status/1426267327687847939)
+- [Brave Search](https://search.brave.com/) is using Svelte
+- [exatorrent](https://github.com/varbhat/exatorrent) is a self-hostable, easy-to-use, lightweight and feature-rich torrent client written in Go and Sveltejs
+- [json2TsTypes](https://github.com/jatinhemnani01/json2TsTypes) is a simple tool which will convert your JSON to Typescript Types/Interfaces
+- [Histogram.dev](https://histogram.dev/) generates histograms for each feature in a CSV
+- [cybernetic.dev](https://cybernetic.dev/) is a collection of data-centric UI experiments made while learning Svelte
+- [LunaNotes](https://chrome.google.com/webstore/detail/lunanotes-youtube-video-n/oehoffnnkgcdacmbkhmlbjedinpampak?hl=en) is a Chrome extension to help with taking YouTube video notes
+- [theia.games](https://theia.games/#dev)'s built-in 3D environment editor lets you create a VR world with a menu built in Svelte
+- [Ferrum](https://github.com/probablykasper/ferrum) is a music library and player available for Mac, Windows or Linux
+- [Fluid Earth](https://github.com/byrd-polar/fluid-earth) is an interactive WebGL application for visualizing Earth's atmosphere and oceans
+
+**Looking for a Svelte project to work on? Interested in helping make Svelte's presence on the web better?** Check out [the list of open issues](https://github.com/svelte-society/sveltesociety-2021/issues) if you'd like to contribute to the Svelte Society rewrite in SvelteKit.
+
+**Educational Content**
+- [Tauri with Standard Svelte or SvelteKit](https://medium.com/@cazanator/tauri-with-standard-svelte-or-sveltekit-ad7f103c37e7) walks through how to setup Svelte with Tauri, a new light-weight framework for developing cross-platform hybrid desktop applications
+- [Svelte - Web App Development Reimagined [An Intro to Svelte]](https://www.youtube.com/watch?v=4CGzFwHoD0A&list=PLEx5khR4g7PKSASVAXXiAhkyx02_OeruP) is a great intro talk from the goto; conference
```

Comment:
will the square brackets inside square brackets render correctly here?

Reply:
Yup! Here it is running in dev mode:
![image](https://user-images.githubusercontent.com/7872348/131588958-77f630a3-3c4a-4940-9baf-02fcff8f113b.png)